### PR TITLE
Do not force puppet:/// schema resource

### PR DESCRIPTION
**Pull request by abednarik**
Related to Issue #462

### CODE REVIEW

**Comment by abednarik** [comment]
I already signed CLA. Please let me know if I need to do something else.

Cheers

File: spec/classes/000_elasticsearch_init_spec.rb [hunk 1]
```diff
@@ -165,6 +165,18 @@
             it { should contain_package('elasticsearch').with(:ensure => 'present', :source => "/opt/elasticsearch/swdl/package.#{pkg_ext}", :provider => "#{pkg_prov}") }
           end
 
+          context 'using file:/// schema' do
```
Comment (electrical) [inline spec/classes/000_elasticsearch_init_spec.rb]: There is an other file:// test already in there below all the others ?

Comment (abednarik) [inline spec/classes/000_elasticsearch_init_spec.rb]: You are right. Should I move this test inside that context, right?

Comment (electrical) [inline spec/classes/000_elasticsearch_init_spec.rb]: I think both tests already do the same thing for the package tests right?

Comment (abednarik) [inline spec/classes/000_elasticsearch_init_spec.rb]: Yes, sorry about that.

**Comment by abednarik** [comment]
I made a new push with this change. It's ok or should I close this PR and made a new one?

Thanks @electrical

**Comment by electrical** [comment]
This is fine, thank you!
as soon as Jenkins is happy i'll merge it

**Comment by electrical** [comment]
jenkins, test it

**Comment by electrical** [comment]
@abednarik would you be able to squash your commits into a single commit? thanks!

**Comment by abednarik** [comment]
Sure, let me see how can I do that :)

**Comment by abednarik** [comment]
Can you please review @electrical ? I made a rebase and then squash my commits into a single one. It's the first time i made this, so maybe wrong. I also notice that i didn't create a feature branch, which is not recommended :(

**Comment by electrical** [comment]
Looks good so far. I'll let Jenkins do its thing and merge it when green.
Again thank you for the contribution!

**Comment by electrical** [comment]
jenkins, test it

**Comment by abednarik** [comment]
Finally!! :)

**Comment by electrical** [comment]
Thank you for your contribution :)

**Pull request merged**